modules/upgrade/partials/incompat-changes.adoc

```diff
@@ -0,0 +1,9 @@
+=== Review incompatible changes
+
+* Starting in version 24.2, when managing configuration properties using the AlterConfigs API directly or the `kafka-configs.sh` shell script without incremental flags, Redpanda now resets all unspecified values to their default values. This aligns more closely with the behavior in Apache Kafka. There is no change if you're managing configuration with tools like `rpk`, Redpanda Console, Kubernetes, Helm, or Terraform. 
++
+NOTE: An exception is made for the `redpanda.remote.` topic properties, such as `redpanda.remote.delete`. The remote properties are intentionally sticky to avoid the possibility of disabling Tiered Storage.
+
```

Comment:
```suggestion
NOTE: An exception occurs for the `redpanda.remote.` topic properties, such as `redpanda.remote.delete`. The remote properties are intentionally sticky to avoid the possibility of disabling Tiered Storage.
```

Reply:
I'd avoid the use of the term "sticky" here--developer slang.